test/test_public_bindings.py

```diff
@@ -133,6 +133,7 @@ def test_no_new_bindings(self):
             "GraphExecutorState",
             "has_cuda",
             "has_cudnn",
+            "has_kleidiai",
```

Comment:
Can we avoid adding this to the top level `torch.` module and use the `torch.backends.kleidiai.is_available()` directly  please.

Reply:
The CI fail here with assertion : https://github.com/pytorch/pytorch/blob/83c594ebd6dfa517fdd67ae23929cc60d5fa325d/test/test_public_bindings.py#L255 without this change

We can add the has_kleidiai to : https://github.com/pytorch/pytorch/blob/83c594ebd6dfa517fdd67ae23929cc60d5fa325d/test/test_public_bindings.py#L245 if you feel thats the right way to go

torch.backends.kleidiai.is_available() uses has_kleidiai check internally

Reply:
https://github.com/pytorch/pytorch/pull/134124/files#diff-0ccbfbf81a5d8848fa3fb17fdb73b88cc65d5004a778d7a131056798e7b80d66R5